build.zig

```diff
@@ -48,6 +48,21 @@ pub fn build(b: *zbs.Builder) !void {
 
     const examples = b.option(bool, "examples", "Set to true to build examples") orelse false;
 
+    const version = v: {
+        const version_string = "0.1.0";
+
+        var ret: u8 = undefined;
```

Comment:
```suggestion
```

Reply:
hmm why? I need it in `execAllowFail()`, or how should I make it?

Reply:
Ah, I missed that this was passed to `execAllowFail()` and thought it was just unused.